### DISCUSSION TITLE
Multiple sources in a single track

### DISCUSSION THREAD

**Discussion by swaptr** [katspaugh/wavesurfer.js#3455] [Q&A] [answered]
I want to use wavesurfer for managing playback for multiple sources of audio as a single track. Also I would like to avoid stitching the audio upfront so i agree its non ideal but is there any community example or workaround?

**Comment by silverDent** [comment]
Hey [katspaugh](https://github.com/katspaugh) , We also have the same use-case of adding multiple sources in single track, can you please helps us with this?
or a small guide to achieve this would be appreciate.

**Reply by katspaugh** [reply]
Can you explain a little bit better what you want?

**Comment by swaptr** [comment]
My use case is similar to this timeline functionality in [Kdenlive](https://kdenlive.org/en/). 
  

![Screenshot_20240101_221822](https://github.com/katspaugh/wavesurfer.js/assets/83858160/435a5db8-7933-44bf-8b3d-ff057b1b54fb)  

It is similar to the multitrack plugin but I want to use it for multiple audio sources in a single track and then multiple tracks for the final overlap enabled render. Does wavesurfer support this natively where I can specify multiple audio sources for a single track possibly by specifying an array of URLs?

**Reply by katspaugh** [reply, marked as answer]
I see. No, it doesn’t support this out of the box.

You could probably mod the multitrack plugin to do this. Perhaps even with just CSS, e.g. a negative margin to move lower tracks up.

**Reply by FSDRahulK** [reply]
Hi @swaptr, were you able to achieve this? Even I'm trying to build something similar, but stuck here.
